### ISSUE TITLE
Tool for managing open windows

### ISSUE THREAD

**Issue by jamesmortensen** [nwutils/nwutils.github.io#9]
I noticed you had a tool for managing open windows on the ideas board.  Several years ago we put together a tool that manages open windows, including passing data to them as well as being able to close them.  

https://github.com/procedurallygenerated/node-webkit-window-manager

Some of the API's may be out of date, but I'm hoping this helps.

**Comment by TheJaredWilcurt** [comment]
Yeah, that's a bit dated. but at least it works. I'll add a link to it. Any interest in updating it?

**Comment by jamesmortensen** [comment]
I will check it out in the latest nwjs SDK and get back to you with an
update.
